packages/synthchron-tauri/src/components/editor/bottomDrawer/postprocessingPanel/PostprocessingPanel.tsx

```diff
@@ -1,76 +1,78 @@
 import React, { useCallback, useEffect, useState } from 'react'
 
-import {
-  DndContext,
-  DragEndEvent,
-  KeyboardSensor,
-  PointerSensor,
-  closestCenter,
-  useSensor,
-  useSensors,
-} from '@dnd-kit/core'
-import {
-  SortableContext,
-  arrayMove,
-  sortableKeyboardCoordinates,
-  verticalListSortingStrategy,
-} from '@dnd-kit/sortable'
-import AddIcon from '@mui/icons-material/Add'
 import HelpIcon from '@mui/icons-material/Help'
-// Temporary
-import { IconButton, Paper, Slider, Tooltip, Typography } from '@mui/material'
+import {
+  Checkbox,
+  Collapse,
+  Divider,
+  IconButton,
+  Paper,
+  Slider,
+  Stack,
+  Tooltip,
+  Typography,
+} from '@mui/material'
 import { debounce } from 'lodash'
 
 import {
   PostprocessingConfiguration,
   PostprocessingStepType,
 } from '@synthchron/types'
 
-import { SortableItem } from './SortableItem'
+import { PostprocessingCheckboxLine } from './PostprocessingCheckboxLine'
 
 const postprocessingText = `Apply postprocessing steps to the generated traces to add noise. 
-The weight chooses which postprocessing step should be applied. 
-Currently, only one type of noise can be applied to each event.`
+The noise probability controls how whether noise should be applied to an event.
```

Comment:
'how whether' is bad
I suggest -> The noise probability controls how much noise should be applied to an event.
or -> The noise probability controls whether noise should be applied to an event.

Reply:
Good catch, will fix